content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -329,7 +329,7 @@ To enable the plugin, configure the following flags on the API server:
 | `--oidc-groups-prefix` | Prefix prepended to group claims to prevent clashes with existing names (such as `system:` groups). For example, the value `oidc:` will create group names like `oidc:engineering` and `oidc:infra`. | `oidc:` | No |
 | `--oidc-required-claim` | A key=value pair that describes a required claim in the ID Token. If set, the claim is verified to be present in the ID Token with a matching value. Repeat this flag to specify multiple claims. | `claim=value` | No |
 | `--oidc-ca-file` | The path to the certificate for the CA that signed your identity provider's web certificate. Defaults to the host's root CAs. | `/etc/kubernetes/ssl/kc-ca.pem` | No |
-| `--oidc-signing-algs` | The signing algorithms accepted. Default is "RS256". | `RS512` | No |
+| `--oidc-signing-algs` | The signing algorithms accepted. Default is "RS256". Allowed values are "RS256" "RS384" "RS512" "ES256" "ES384" "ES512" "PS256" "PS384" "PS512". | `RS512` | No |
```

Comment:
How I'd write this:
```suggestion
| `--oidc-signing-algs` | The signing algorithms accepted. Default is RS256. Allowed values are: RS256, RS384, RS512, ES256, ES384, ES512, PS256, PS384, PS512. | `RS512` | No |
```

Extra credit: make it clear how to specify multiple accepted algorithms.

Reply:
Thanks, I also added a reference to the RFC: I found out that the algorithms were already documented under `--oidc-signing-algs strings` in the kube-apiserver options documentation:
https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/#options